integration/data/test_basic_ops.py

```diff
@@ -42,31 +44,35 @@ def test_basic_rw(dev):  # NOQA
         verify_data(dev, offset, data)
 
 
-def test_rw_with_metric(grpc_controller,  # NOQA
-                        grpc_replica1, grpc_replica2):  # NOQA
-    rw_dev = get_dev(grpc_replica1, grpc_replica2, grpc_controller)
+def test_rw_with_metric(grpc_replica1, grpc_replica2, grpc_controller):  # NOQA
+    rw_dev = get_dev(grpc_replica1, grpc_replica2,
+                     grpc_controller)
 
-    replies = grpc_controller.metric_get()
-    # skip the first metric since its fields are 0
-    next(replies).metric
-
-    for i in range(0, 5):
+    for i in range(0, 1):
```

Comment:
Why need two iteration?

Reply:
No, I mean what is the original case having 5 iterations?

Reply:
Well...I think just the first iteration is to check the previous grpc streaming works as expected. It can be removed here, because we don't use streaming here.

The second iteration is due to that the metrics are not updated yet.

Reply:
Updated the test case.